services/brig/src/Brig/API/Client.hs

```diff
@@ -209,6 +202,13 @@ claimLocalPrekeyBundles = foldMap getChunk . fmap Map.fromList . chunksOf 16
       when (isNothing key) $ noPrekeys u c
       return key
 
+-- FUTUREWORK: delete this in favor of claimMultiPrekeyBundles
```

Comment:
I don't quite understand at which point this function should be deleted. What needs to change for this to become possible? Can it be done in this PR? If not, what needs to be in place for it to be done?

Reply:
This should have been a TODO instead of FUTUREWORK. I removed it. `claimMultiPrekeyBundlesLocal` made code more DRY, so I left it